src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/tests/Action.test.js

```diff
@@ -19,24 +19,6 @@ test('The component should call the callbacks after a click', () => {
     expect(afterAction).toBeCalled();
 });
 
-test('The component should call the callbacks after press enter', () => {
```

Comment:
Can we adjust the tests that we are sure this still works?

Reply:
The Enter and Space keys trigger the click event for buttons, so an `EventListener` listening for keypresses is not triggered. An explicit test for this scenario may not be necessary, as it ultimately only tests the function of the native button element.

Reply:
Make sense!